parm/soca/obs/obs_list.yaml

```diff
@@ -1,5 +1,17 @@
 observers:
+# ADT
 - !INC ${OBS_YAML_DIR}/adt_rads_all.yaml
+
+# SST
+- !INC ${OBS_YAML_DIR}/sst_metopa_l3u.yaml
```

Comment:
@guillaumevernieres, if you concur with the previous naming suggestion, these need to be changed and `metop<b,c>` as well.